neurodsp/spectral/power.py

```diff
@@ -241,3 +244,47 @@ def compute_spectrum_medfilt(sig, fs, filt_len=1., f_range=None):
         freqs, spectrum = trim_spectrum(freqs, spectrum, f_range)
 
     return freqs, spectrum
+
+
+def compute_spectrum_multitaper(sig, fs, bandwidth, num_windows):
```

Comment:
this func can be aliased in `spectral/__init__.py` so it can be directly imported from neurodsp.spectral like the other funcs

Reply:
Done! Should work now.